src/Negotiation/Negotiator.php

```diff
@@ -21,23 +21,62 @@ protected function match(AcceptHeader $accept, AcceptHeader $priority, $index)
             return null;
         }
 
-        $ab = $accept->getBasePart();
-        $pb = $priority->getBasePart();
+        $acceptBase = $accept->getBasePart();
+        $priorityBase = $priority->getBasePart();
```

Comment:
I changed these variable names as I was confused by the two-character names when attempting to make my changes. I can revert these if desired, but I think they make the intent far clearer.